code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -1878,70 +1962,270 @@
 	icon_state = "accessory_none"
+	marking_location = "body"
 
 /datum/sprite_accessory/body_markings/none
 	name = "None"
 	species_allowed = list("Human", "Unathi", "Diona", "Grey", "Machine", "Tajaran", "Vulpkanin", "Slime People", "Skeleton", "Vox")
 	icon_state = "accessory_none"
 
-/datum/sprite_accessory/body_markings/stripe
-	name = "Stripe"
-	species_allowed = list("Unathi")
-	icon_state = "markings_stripe"
-
 /datum/sprite_accessory/body_markings/tiger
 	name = "Tiger Body"
 	species_allowed = list("Unathi", "Tajaran", "Vulpkanin")
 	icon_state = "markings_tiger"
 
-/datum/sprite_accessory/body_markings/tigerhead
-	name = "Tiger Body and Head"
-	species_allowed = list("Unathi", "Tajaran", "Vulpkanin")
-	icon_state = "markings_tigerhead"
+/datum/sprite_accessory/body_markings/stripe_una
+	name = "Unathi Stripe"
+	species_allowed = list("Unathi")
+	icon_state = "markings_stripe_una"
+
+/datum/sprite_accessory/body_markings/belly_narrow_una
+	name = "Unathi Belly"
+	species_allowed = list("Unathi")
+	icon_state = "markings_belly_narrow_una"
+
+/datum/sprite_accessory/body_markings/banded_una
+	name = "Unathi Banded"
+	species_allowed = list("Unathi")
+	icon_state = "markings_banded_una"
+
+/datum/sprite_accessory/body_markings/points_una
+	name = "Unathi Points"
+	species_allowed = list("Unathi")
+	icon_state = "markings_points_una"
+
+/datum/sprite_accessory/body_markings/belly_flat_taj
+	name = "Tajaran Belly"
+	species_allowed = list("Tajaran")
+	icon_state = "markings_belly_flat_taj"
+
+/datum/sprite_accessory/body_markings/belly_crest_taj
+	name = "Tajaran Chest Crest"
+	species_allowed = list("Tajaran")
+	icon_state = "markings_belly_crest_taj"
+
+/datum/sprite_accessory/body_markings/belly_full_taj
+	name = "Tajaran Belly 2"
+	species_allowed = list("Tajaran")
+	icon_state = "markings_belly_full_taj"
+
+/datum/sprite_accessory/body_markings/points_taj
+	name = "Tajaran Points"
+	species_allowed = list("Tajaran")
+	icon_state = "markings_points_taj"
+
+/datum/sprite_accessory/body_markings/patchy_taj
+	name = "Tajaran Patches"
+	species_allowed = list("Tajaran")
+	icon_state = "markings_patch_taj"
+
+/datum/sprite_accessory/body_markings/belly_fox_vulp
+	name = "Vulpkanin Belly"
+	species_allowed = list("Vulpkanin")
+	icon_state = "markings_belly_fox_vulp"
+
+/datum/sprite_accessory/body_markings/belly_full_vulp
+	name = "Vulpkanin Belly 2"
+	species_allowed = list("Vulpkanin")
+	icon_state = "markings_belly_full_vulp"
+
+/datum/sprite_accessory/body_markings/belly_crest_vulp
+	name = "Vulpkanin Belly Crest"
+	species_allowed = list("Vulpkanin")
+	icon_state = "markings_belly_crest_vulp"
+
+/datum/sprite_accessory/body_markings/points_fade_vulp
+	name = "Vulpkanin Points"
+	species_allowed = list("Vulpkanin")
+	icon_state = "markings_points_fade_vulp"
+
+/datum/sprite_accessory/body_markings/points_fade_belly_vulp
+	name = "Vulpkanin Points and Belly"
+	species_allowed = list("Vulpkanin")

```

Comment:
Sounds like a good idea to me, thanks for the detail. I'd like to leave that for another PR that squares it all away at once. Think there'd be any performance advantage that would come along with such a change, too?